public/static/js/patch_new.js

```diff
@@ -91,30 +92,37 @@ mciModule.controller('PatchController', function($scope, $filter, $window, notif
     }
   }
 
+  $scope.isUnauthorizedPRPatch = (patch.Version.length === 0 && patch.GithubPatchData.PRNumber !== 0);
+  $scope.isPRPatch = patch.GithubPatchData.PRNumber !== 0;
+
+  $scope.collectVariantTasks = function() {
```

Comment:
I don't see that this is called outside of the control so I wouldn't put this in $scope